tests/dashboard/test_permissions.py

```diff
@@ -1032,3 +1033,61 @@ def test_only_staff_user_with_permission_can_update_payment_gateway(
     staff_user.user_permissions.add(permission_manage_plugins)
     response = staff_client.get(url)
     assert response.status_code == 200
+
+
+@pytest.mark.parametrize(
+    "url_name", ["dashboard:warehouse-list", "dashboard:warehouse-create"]
```

Comment:
We are dropping dashboard soon - not sure if we should add new tests and views in the meantime.

Reply:
This PR will be also added to legacy dashboard